features/install-cli.feature

```diff
@@ -52,6 +54,10 @@ Feature: Command behavior of client side tools installation
     Then the exit status should be 0
     And the binary "docker" should be installed
 
+    When I run `bundle exec vagrant service-manager install-cli docker --cli-version 1.12.1`
```

Comment:
but that's just the plain _install-cli_, right? I believe in this case we don't have the _--path_ and _--version_ option, right? Something we should test to verify the expected error message occurs. On the adb feature file we still can then test the 'oc' download with path and version.